docs/developer_guide/SimpleSensorSimulator.md

```diff
@@ -2,25 +2,98 @@
+| horizontal_resolution | double | radian | Horizontal resolutions of the lidar               |
+| vertical_angles       | double | radian | Vertical resolutions of the lidar                 |
+| scan_duration         | double | second | Scan duration of the lidar                        |
+| architecture_type     | string | -      | Autoware architecture type                        |
+
+### Acknowledgments
 In lidar simulation, we use intel's ray-casting library embree.
 
-<iframe 
-  class="hatenablogcard" 
-  style="width:100%;height:155px;max-width:450px;" 
-  title="embree" 
-  src="https://hatenablog-parts.com/embed?url=https://www.embree.org" 
-  width="300" height="150" frameborder="0" scrolling="no">
+<iframe
+class="hatenablogcard"
+style="width:100%;height:155px;max-width:450px;"
+title="embree"
+src="https://hatenablog-parts.com/embed?url=https://github.com/embree/embree"
+width="300" height="150" frameborder="0" scrolling="no">
 </iframe>
 
-<iframe 
-  class="hatenablogcard" 
-  style="width:100%;height:155px;max-width:450px;" 
-  title="embree" 
-  src="https://hatenablog-parts.com/embed?url=https://github.com/embree/embree" 
-  width="300" height="150" frameborder="0" scrolling="no">
-</iframe>
+## Occupancy Grid Sensor Simulation
+With this simulation, we can get a cost map without processing lidar point cloud data.  
+This enables us to reduce computational resources when we want to test Autoware's planners or controllers.  
+
+### Interfaces
+
+| interface                                   | type                                   | note                                                                                       |
+|---------------------------------------------|----------------------------------------|--------------------------------------------------------------------------------------------|
+| `traffic_simulator::API::attachOccupancyGridSensor` | C++ traffic simulator API interface    |                                                                                            |
+| `attach_occupancy_grid_sensor`                       | ZeroMQ traffic simulator API interface | See [ZeroMQ Interfaces documentation](/docs/developer_guide/ZeroMQ.md)<br/>TCP Port : 5565 |  
+
+### Occupancy Grid Values
+
+| grid type      | value | description                                                         |
+|----------------|-------|---------------------------------------------------------------------|
+| occupied grid  | 100   | a grid determined by a simulated lidar sensor that an object exists |
+| empty grid     | 0     | an empty grid proved by a simulated ray-cast passing through        |    
+| invisible grid | 50    | a grid that is out of range or occlusion of simulated lidar sensor  |
+
+
+### Configuration
+| name              | type   | unit   | description                                                                                                                         |
```

Comment:
Please remove this table.
Configuration should be auto-generated from protobuf-gen-c-doc